nodes/test/test_chroma_filter_semantics.py

```
@@ -0,0 +1,145 @@
+from __future__ import annotations
+
+import importlib.util
+import sys
+import types
+from pathlib import Path
+
+
+def _install_stubs() -> None:
+    """Install lightweight stubs so chroma.py can be imported in isolation."""
+    mod_depends = types.ModuleType('depends')
+    mod_depends.depends = lambda *args, **kwargs: None
```

Ruff (0.15.6)

[warning] Unused lambda argument: `args`

(ARG005)